package.json

```diff
@@ -7,12 +7,15 @@
   "devDependencies": {
     "gulp": "^3.9.0",
     "gulp-karma": "0.0.1",
+    "jasmine-core": "^2.8.0",
     "karma": "^0.13.0",
     "karma-chrome-launcher": "^0.2.0",
     "karma-coverage": "latest",
-    "karma-firefox-launcher": "latest",
+    "karma-phantomjs-launcher": "latest",
```

Comment:
Putting "latest" here and for the package below worries me because a breaking change could be released for these packages and break our builds. Using the carrot will upgrade everything except new major versions, which is what I think we want.

Reply:
Roger, I'll update before merging